src/app.md

```diff
@@ -234,6 +235,33 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-sizes]: #ex-file-sizes
+<a name="ex-file-sizes"></a>
+## Recursively calculate file sizes at given depth
+
+min_depth & max_depth usage example to calculate total file size at given depth.
```

Comment:
Please "linkify" the mentioned symbols ([`WalkDir::min_depth`](https://docs.rs/walkdir/1.0.7/walkdir/struct.WalkDir.html#method.min_depth) and [`WalkDir::max_depth`](https://docs.rs/walkdir/1.0.7/walkdir/struct.WalkDir.html#method.max_depth)) .
The source code of [following recipe](http://localhost:3000/net.html#remove-fragment-identifiers-and-query-pairs-from-a-url) is a good example.